teuthology/provision/test/test_init_provision.py

```diff
@@ -0,0 +1,46 @@
+from copy import deepcopy
+from pytest import raises
+from teuthology.config import config
+
+import teuthology.provision
+
+test_config = dict(
+    pelagos=dict(
+    endpoint='http://pelagos.example:5000/',
+    machine_types='ptype1,ptype2,common_type',
+    ),
+    fog=dict(
+    endpoint='http://fog.example.com/fog',
+    api_token='API_TOKEN',
+    user_token='USER_TOKEN',
+    machine_types='ftype1,ftype2,common_type',
+    )
+)
+
+class TestInitProvision(object):
+
+    def setup(self):
+        config.load(deepcopy(test_config))
+
+    def test_get_reimage_types(self):
+        reimage_types = teuthology.provision.get_reimage_types()
+        assert reimage_types == ["ptype1","ptype2", 'common_type',
+                                "ftype1","ftype2", 'common_type']
```

Comment:
(style) put space after comma

Reply:
fixed

Reply:
Not exactly, could you please also use the same quotes for each element of list?

Reply:
and is it me or there are two of 'common_type' elements?

Reply:
done